src/store.js

```diff
@@ -196,6 +196,11 @@ class AsyncStorageStore {
       {
         pin,
         password: pin,
+        // NOTE:
+        // This NETWORK usage doesn't need to get the network
+        // from the networkSettings in the redux steore,
+        // because this one is used in the context of wallet
```

Comment:
This is not wallet initialization, because in the initialization we need the correct network, to generate the correct addresses.

This is the access data generation.

Reply:
- Updated comment: b666252ff348bafb01e1bcc6e68b5539e1c09c20